QCflow/run_opt_neutral.py

```diff
@@ -37,3 +37,34 @@ def run_opt_neutral(mol_name, mol_smile, mol_dic, functional='B3LYP', basis_set=
     submit_slurm_job('pop_opt_n', mol_name)
     #goes back to previous directory
     os.chdir(os.path.dirname(os.getcwd()))
+
+def run_opt_neutral_set_dihedral(mol_name, mol_smile, functional='B3LYP', basis_set='6-31G*', dihedral_angle=0):
+    """
+    Submits a neutral optimization with population analysis when provided with the name of the mol
+
+    mol_name : name of oligomer
+
+    mol_smile : SMILE string of oligomer
+
+    functional : preset is B3LYP
+
+    basis_set : preset is 6-31G*
+
+    dihedral_angle : Angle to set dihedral to (preset is 0)
+    """
+    #goes into directory
+    os.chdir(f'{mol_name}')
+    #turns smiles string into rdkit object
+    mol = Chem.MolFromSmiles(mol_smile)
+    #gets rdkit estimated coordinates of dimer
+    mol3d = embed_molecule(mol)
+    #Saves the torsional scan as .csv and finds the lowest energy geometry
+    conf_geo = setting_dihedral(mol_smile, dihedral_angle)
+    #writes a guassian input file
+    write_gaussian('pop_opt_n', mol_name, mol_smile, functional, basis_set, mol3d, 0, conf_geo)
+    #writes the slurm file
+    write_slurm('pop_opt_n', mol_name)
+    #submits the slurm jon
+    submit_slurm_job('pop_opt_n', mol_name)
+    #goes back to previous directory
+    os.chdir(os.path.dirname(os.getcwd()))
```

Comment:
Allows the user to set the dihedral angle of there dimer. This is good if you already know what the dihedral is or want it to be. I have never tested it on trimers but this could potentially work or be adapted for Hamza

Reply:
this feels out of scope - better to write a `set_dihedral` function separately

Reply:
remove - you have a redundant function set_dihedral already

Reply:
I will move this to the scripts folder instead as I think this is still useful for that. With the view it moves to an exemplar of how the code can be utilised